quetz/main.py

```diff
@@ -37,36 +37,46 @@
 )
 from quetz.config import Config
 from quetz.dao import Dao
-from quetz.database import get_session as get_db_session
 
 from .condainfo import CondaInfo
 
 app = FastAPI()
 
-config = Config()
-auth_github.register(config)
+if "QUETZ_IS_TEST" in os.environ:
```

Comment:
used for testing with pytest?
why not creating an `app` fixture with this configuration instead?

I can't see the `QUETZ_IS_TEST` env var in the CI. Are the tests passing?

Reply:
The problem is that (right now) we need the app object in this file to attach all routes to it.
However, when we create routers, we can attach them to the app object and refactor, which will be much nicer (and allow us to create the fixture). I'll take care of that today.